torch/serialization.py

```diff
@@ -29,6 +29,21 @@
 PROTOCOL_VERSION = 1001
 STORAGE_KEY_SEPARATOR = ','
 
+__all__ = [
```

Comment:
This came up in #74171 too. The `test_correct_module_names` test is very fragile, and any new import technically changes the public API unless `__all__` is defined. Once again, I randomly chose to list all classes/methods that don't start with an underscore, but let me know if anything should be added/removed in `__all__`.